TYPO3.Neos/Resources/Public/JavaScript/Content/Model/Node.js

```diff
@@ -203,7 +203,7 @@ define([
 				label = (this.get('nodeTypeLabel') || this.get('nodeType')) + ' (' + this.getAttribute('_name') + ')';
 			}
 
-			label = $('<span>' + label + '</span>').text().trim();
+			label = $('<u/>').html(label).text().trim();
```

Comment:
yeah, have made that mistake before where it sometimes worked and not other times since the input was lacking a wrapping tag..